<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageExplorer</class>
 <widget class="QMainWindow" name="ImageExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Image Explorer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>432</width>
     <height>243</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>960</width>
        <height>540</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="treeView">
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>81</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>320</width>
         <height>180</height>
        </size>
       </property>
      </widget>
      <widget class="ImageList" name="imListView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>81</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>320</width>
         <height>180</height>
        </size>
       </property>
      </widget>
      <widget class="ImageFrame" name="imageFrame">
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>81</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>320</width>
         <height>180</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="imageLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>144</width>
            <height>81</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>320</width>
            <height>180</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageFrame</class>
   <extends>QFrame</extends>
   <header>imageframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageList</class>
   <extends>QListView</extends>
   <header>imagelist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
